front/app/containers/Admin/projects/project/analysis/Taggings/index.tsx

```diff
@@ -26,8 +26,12 @@ const Taggings = ({
 
   const { data: taggings } = useAnalysisTaggings(analysisId);
 
-  const taggingsForInput = taggings?.data?.filter(
-    (tagging) => tagging.relationships.input.data.id === inputId
+  const taggingsForInput = useMemo(
```

Comment:
Not a huge impact but arguably for bigger numbers it can be beneficial to memoize this.